content/no/template.json

```diff
@@ -1,14 +1,55 @@
 {
-  "browser-title":"io.js - JavaScript I/O",
-  "logo-text":"io.js",
+  "browser-title": "io.js - JavaScript I/O",
+  "contribute-message": "Ser du noe du liker? Vil du hjelpe til? Gå til https://github.com/nodejs/website for å bidra",
+  "heading-languages": "Språk",
+  "logo-text": "io.js",
   "roadmap-url": "http://roadmap.iojs.org",
   "roadmap-link": "Roadmap",
-  "faq-link":"FAQ",
-  "es6-link":"ES6",
-  "api-link":"API",
-  "issues-link":"GitHub Issues",
-  "org-link":"GitHub Org",
-  "irc-link":"IRC Chat",
-  "irc-logs-link":"Historikk",
-  "gov-link":"Styringsmodell"
+  "faq-link": "FAQ",
+  "es6-link": "ES6",
+  "api-link": "API Docs",
+  "issues-link": "GitHub Issues",
+  "org-link": "GitHub Org",
+  "irc-link": "IRC Chat",
```

Comment:
While I agree, this is the text in the original english file (and in the currently used Norwegian file). IRC is also the name of the protocol, so in one way one can argue that it describes the type of protocol used in the chat.

Anyway... We could translate this I guess. `IRC kanal`?